assets/src/edit-story/components/panels/panel/shared/title.js

```diff
@@ -32,8 +32,10 @@ import {
   Icons,
   THEME_CONSTANTS,
   Headline,
+  themeHelpers,
+  ThemeGlobals,
 } from '../../../../../design-system';
-import { KEYBOARD_USER_SELECTOR } from '../../../../utils/keyboardOnlyOutline';
+import { BUTTON_TRANSITION_TIMING } from '../../../../../design-system/components/button/constants';
```

Comment:
Should we move this to general constants in the design system perhaps?

Reply:
Yeah this is an awkward import 😅, I changed this in a different pr too. I ended up doing this: https://github.com/google/web-stories-wp/pull/6786/files#diff-bc0076673501c056995205221a976b39f5fb74b1febffaa3a8f9c5195356eb60R18

Reply:
It looks like `export * from './button';` is exporting constants that are also exported by `export * from './constants';` so there would actually be a collision of exports this way. Maybe we should export it from the `/button` directly instead. Thoughts?

Reply:
oh no! yeah i guess exporting from button is the best way to do this then.

Reply:
Up to you! we'd have to change it in a few places for it to work:
![image](https://user-images.githubusercontent.com/22185279/111817623-d59fd280-88a3-11eb-9680-ea5bef91de74.png)

My preference is to export constants from their own file but I think either is fine.